### ISSUE TITLE
Export in Pascal VOC format

### ISSUE THREAD

**Issue by jsbroks** [jsbroks/coco-annotator#36]
Support for exporting in Pascal VOC format ([example](https://gist.github.com/Prasad9/30900b0ef1375cc7385f4d85135fdb44))

**Comment by MentalGear** [comment]
Hi there, has there been made progress on this already ?

**Comment by jsbroks** [comment]
@MentalGear Yes, I created [imantics](https://github.com/jsbroks/imantics) to resolve this but I have not added UI component to access the API though the webclient
